modules/appeals_api/app/models/appeals_api/higher_level_review.rb

```diff
@@ -7,6 +7,10 @@ module AppealsApi
   class HigherLevelReview < ApplicationRecord
     include HlrStatus
 
+    scope :pii_expunge_policy, lambda {
+      where('updated_at < ? AND status IN (?)', 7.days.ago, COMPLETE_STATUSES)
```

Comment:
I definitely prefer this iteration, I think the main idea is leveraging the tools available, and promoting readability. What about the NOD version do you dislike? I think it's a pretty readable subquery now.